quarkus/runtime/pom.xml

```diff
@@ -91,7 +91,6 @@
         <dependency>
             <groupId>org.wildfly.security</groupId>
             <artifactId>wildfly-elytron</artifactId>
-            <version>${wildfly-elytron.version}</version>
```

Comment:
I assume that at least when it comes to Quarkus module we don't need a version elements specified here at all, right? Since a dependency is part of the `<dependencyManagement>` in the parent pom, we can omit it in its children.

Reply:
Yes, we can remove it.